pkg/app/run.go

```diff
@@ -245,7 +151,7 @@ func (run *Run) diff(triggerCleanupEvent bool, detailedExitCode bool, c DiffConf
 	for _, r := range changedReleases {
 		id := state.ReleaseToID(&r)
 
-		// If `helm-diff` detected changes but it is not being `helm delete`ed, we should run `helm upgrade`
+		// If `helm-diff` detected changes but it is not being `helm delete`ed, we should r `helm upgrade`
```

Comment:
```suggestion
		// If `helm-diff` detected changes but it is not being `helm delete`ed, we should run `helm upgrade`
```

Seems like an unnecessary change 👍